src/vivarium_gates_nutrition_optimization_child/components/causes.py

```diff
@@ -1,17 +1,81 @@
 """
 Component to include birth prevalence in SIS model.
 """
-
+import pandas as pd
 from vivarium.framework.state_machine import State
-from vivarium_public_health.disease import DiseaseModel, DiseaseState
+from vivarium_public_health.disease import DiseaseModel as DiseaseModel_
+from vivarium_public_health.disease import DiseaseState
 from vivarium_public_health.disease import (
     RiskAttributableDisease as RiskAttributableDisease_,
 )
 from vivarium_public_health.disease import SusceptibleState
 from vivarium_public_health.disease.transition import TransitionString
 
 
-def SIS_with_birth_prevalence(cause: str) -> DiseaseModel:
+class DiseaseModel(DiseaseModel_):
+    @property
+    def columns_required(self):
```

Comment:
Are these not already defined in the super DiseaseModel?

Reply:
cf. Rajan's comment, I added two columns. It's clearer to make the call explicitly to super.columns_required